### PR TITLE
[PR] Home task1 using babel

### PR DESCRIPTION
**Pull request by rakesh-vardan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon --exec babel-node home-task1.1.js"
```
Comment (RinkyDewangan) [inline package.json]: Add scripts also to run task1 and task2 as npm run task1 and npm run task2 respectively

File: home-task1.2_Stream.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+import csv from 'csvtojson';
 
 const CSV_FILE_PATH = 'csv/nodejs-hw1-ex1.csv';
 const OUTPUT_FILE_NAME = 'output.txt';
 
-const readStream = fs.createReadStream(CSV_FILE_PATH).setEncoding('utf-8');
-const writeStream = fs.createWriteStream(OUTPUT_FILE_NAME);
+const readStream = createReadStream(CSV_FILE_PATH).setEncoding('utf-8');
+const writeStream = createWriteStream(OUTPUT_FILE_NAME);
 readStream.pipe(csv()).pipe(writeStream);
```
Comment (RinkyDewangan) [inline home-task1.2_Stream.js]: you also have to use pipeline to prevent loading contents of file fully into RAM.

Comment (RinkyDewangan) [reply home-task1.2_Stream.js]: Sorry this is fine . My bad. Partial loading is okay for the task. There are ways where we can save more memory.

File: package.json (hunk 1 above)
Comment (rakesh-vardan) [reply package.json]: Added as suggested

**Pull request merged**